packages/eyes/src/eyes-test/eyes.ts

```diff
@@ -0,0 +1,168 @@
+const {TestFailedError, TestResults} = require('@applitools/eyes.sdk.core');
+const {makeVisualGridClient, initConfig} = require('@applitools/visual-grid-client');
+const domNodesToCdt = require('@applitools/visual-grid-client/src/browser-util/domNodesToCdt');
+import * as path from 'path';
+import * as fs from 'fs';
+import glob from 'glob';
+import chalk from 'chalk';
+import {JSDOM} from 'jsdom';
+import {consoleLog} from '@ui-autotools/utils';
+import {parseFilename} from '../generate-snapshots/filename-utils';
```

Comment:
I'd call it `parseSnapshotFilename`. `parseFilename` sounds like something that parses a filename into basename and extension.